lambdas/indexer/.chalice/config.json.template.py

```diff
@@ -24,10 +24,14 @@
                 },
                 "write": {
                     "reserved_concurrency": config.indexer_concurrency,
-                    # FIXME: Use a two-stage pipeline with a high memory size 2nd
-                    #        stage to process tallies that failed in the lower
-                    #        size 1st stage. https://github.com/DataBiosphere/azul/issues/1732
-                    "lambda_memory_size": 3008,
+                    "lambda_memory_size": 256,
+                    # TODO: Consider lowering the lambda timeout and write queue
```

Comment:
Please create ticket and remove comment. I am not sure I want to fast-track big tasks.

Reply:
https://github.com/DataBiosphere/azul/issues/1801